pages/2017/index.rst

```diff
@@ -8,4 +8,60 @@ Python in Astronomy 2017
 Lorentz Center, Leiden
 ----------------------
 
-WOOOO YEAG
+The third edition of the Python in Astronomy Conference will be held at the
+Lorentz Center on the **8 - 12th May 2017**.
+
+As with the two previous Python in Astronomy conferences we aim to bring a wide
+variety of people who use, develop or teach people about Python packages in the
+context of all forms of Astronomy. The conference will include presentations,
+tutorials unconference sessions and sprints. As well as building the community
+around astronomical uses of Python the conference aims to improve collaboration
+and interoperability between Python packages, share knowledge on Python packages
+and techniques and provide training and educational materials for users and
+developers of Python packages.
+
+Attending PyAstro17
+###################
+
+Between Sept 12, 2016 and Dec 9, 2016, we will be accepting requests from
+interested participants. Participant selection will be made with the goal of
+*growing* the Python in Astronomy community and we particularly encourage
+requests to attend from junior astronomers and people who are new to
+contributing to open source software. Effort will also be made to select
+participants who have a demonstrated potential to contribute meaningfully to the
+Python in Astronomy infrastructure via providing educational materials,
+documentation, and/or coding. Only basic familiarity with Python is expected of
+participants. This conference is neither intended to be an introduction to
+Python bootcamp nor only for expert-level Python developers.
+
+Until then you can register your interest or nominate someone who
+you feel should attend using the following form:
```

Comment:
In the form, maybe for nominations you should have a 'reason' field or ask people to specify why (so people can say e.g 'they would give an awesome talk on ...'